.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: c
```

Comment:
Shouldn't this be `python`? Just curious since all my other projects use `python`